packages/tables/resources/views/components/pagination/records-per-page-selector.blade.php

```diff
@@ -8,7 +8,11 @@
         'dark:text-white dark:bg-gray-700 dark:border-gray-600 dark:focus:border-primary-600' => config('tables.dark_mode'),
     ])>
         @foreach ($options as $option)
-            <option value="{{ $option }}">{{ $option }}</option>
+            @if ($option === -1)
+                <option value="{{ $option }}">{{ __('tables::table.pagination.all') }}</option>
+            @else
+                <option value="{{ $option }}">{{ $option }}</option>
```

Comment:
We can remove this condition and just echo out `{{ $option ?? __('tables::table.pagination.all') }}` if we use `null` for "all".